packages/@core.next/intl.jsx

```diff
@@ -41,23 +39,14 @@ let getLocale = () => {
     return locale || defaultLocale
 }
 
-let extractReqLocale = (req) => {
```

Comment:
Please leave the default function here

Reply:
There will be naming conflict 😓

Reply:
you can override it as `messagesImporter`